src/app/Components/Artist/ArtistAbout/ArtistAbout.tests.tsx

```diff
@@ -57,4 +57,25 @@ describe("ArtistAbout", () => {
       expect(screen.UNSAFE_queryByType(ArtistAboutShowsFragmentContainer)).toBeTruthy()
     })
   })
+
+  describe("ArtistAboutEditorial", () => {
+    it("renders editorial section", () => {
+      renderWithRelay({
+        ArticleConnection: () => ({
+          edges: [{ node: { internalID: "article-id" } }],
+        }),
+        Artist: () => ({ name: "Andy Warhol" }),
+      })
+
+      expect(screen.getByText("Artsy Editorial Featuring Andy Warhol")).toBeTruthy()
+    })
+
+    it("does not render when there are no articles", () => {
+      renderWithRelay({
+        ArticleConnection: () => ({ edges: null }),
+      })
+
+      expect(screen.queryByText("Artsy Editorial Featuring", { exact: false })).toBeNull()
```

Comment:
**question:** any reason why we need `{ exact: false }` here?

Reply:
to be more false positive proof, in case someone changes something and instead of `Andy Warhol` uses `mocked-data-blah`, with this we're really safe to check when we render or not

Reply:
oh gotcha! Then maybe its better to use due to this reference in the library [docs](https://callstack.github.io/react-native-testing-library/docs/api-queries#precision)


```suggestion
      expect(screen.queryByText(/Artsy Editorial Featuring/)).not.toBeOnTheScreen()
```

(the TLDR; of it is: `In most cases using a regex instead of a string gives you more control over fuzzy matching and should be preferred over { exact: false }.`)

Reply:
done in https://github.com/artsy/eigen/pull/8961/commits/e735d02ca008d8505893020ed177b41abb303073